### PR TITLE
raise error if we cannot decode the manifest as json

### PR DESCRIPTION
**Pull request by jfchevrette**

### CODE REVIEW

File: sretoolbox/container/image.py [hunk 1]
```diff
@@ -390,7 +396,13 @@ def manifest(self):
         """
         if self._cache_manifest is None:
             manifest = self._get_manifest()
-            self._cache_manifest = manifest.json()
+            try:
+                self._cache_manifest = manifest.json()
+            except json.decoder.JSONDecodeError as exc:
+                raise ImageInvalidManifestError(
```
Comment (apahim) [inline sretoolbox/container/image.py:402]: Do we know why is that happening? I see we have the response, but only the JSON conversion is erroing... Is it like incomplete/broken?

**Review by apahim — APPROVED** [review]
lgtm

File: sretoolbox/container/image.py (hunk 1 above)
Comment (apahim) [reply sretoolbox/container/image.py:402]: just curious :)

Comment (apahim) [reply sretoolbox/container/image.py:402]: got it, thanks!

**Review by geoberle — APPROVED** [review]

**Review by apahim — APPROVED** [review]
lgtm, nice test!

**Pull request merged**